api/nakadi-event-bus-api.yaml

```diff
@@ -843,6 +864,9 @@ definitions:
               from the header into the metadata by Nakadi to avoid clients having to explicitly copy this.
             type: string
             example: 'JAh6xH4OQhCJ9PutIV_RYw'
+        required:
+          - eid
+          - occurred_at
```

Comment:
This now conflicts with description of these two fields. Because the description says that these fields are optional.